plugins/enthusiast-common/enthusiast_common/llm/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .base import BaseLLM  # noqa
+from .base import BaseLanguageModelRegistry  # noqa
```

Comment:
Should it be exported by this package, given that base.py then imports it from ..registry?